### ISSUE TITLE
Invalid type for parameter tags - type: <class 'list'>, valid types: <class 'dict'>

### ISSUE THREAD

**Issue by hikarisakamoto** [localstack/localstack#1723]
Hey guys!
I'm having a problem when running a cloud formation script on local stack. Can't find a solution anywhere, and tried a lot different approaches, but still could not make it work.
Is there a different way to declare this declare this values? (Trying to add SQS)

```
2019-11-05T14:38:26:WARNING:localstack.utils.cloudformation.template_deployer: Error calling <bound method ClientCreator._create_api_method.<locals>._api_call of <botocore.client.SQS object at 0x7f3d9ab15a58>> with params: {'QueueName': 'A-DLQ', 'Attributes': {'MessageRetentionPeriod': '1209600'}, 'tags': [{'Key': 'Name', 'Value': 'A-DLQ'}, {'Key': 'Cluster', 'Value': 'A'}]} for resource: {'Type': 'AWS::SQS::Queue', 'Properties': {'QueueName': 'A-DLQ', 'MessageRetentionPeriod': 1209600, 'Tags': [{'Key': 'Name', 'Value': 'A-DLQ'}, {'Key': 'Cluster', 'Value': 'AEvents'}]}}
localstack_1  | 2019-11-05T14:38:26:ERROR:localstack.services.cloudformation.cloudformation_starter: Unable to parse and create resource "DLQ": Parameter validation failed:
localstack_1  | Invalid type for parameter tags, value: [{'Key': 'Name', 'Value': 'A-DLQ'}, {'Key': 'Cluster', 'Value': 'A'}], type: <class 'list'>, valid types: <class 'dict'> Traceback (most recent call last):
localstack_1  |   File "/opt/code/localstack/localstack/services/cloudformation/cloudformation_starter.py", line 149, in parse_and_create_resource
localstack_1  |     return _parse_and_create_resource(logical_id, resource_json, resources_map, region_name)
localstack_1  |   File "/opt/code/localstack/localstack/services/cloudformation/cloudformation_starter.py", line 208, in _parse_and_create_resource
localstack_1  |     result = deploy_func(logical_id, resource_wrapped, stack_name=stack_name)
localstack_1  |   File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 647, in deploy_resource
localstack_1  |     raise e
localstack_1  |   File "/opt/code/localstack/localstack/utils/cloudformation/template_deployer.py", line 644, in deploy_resource
localstack_1  |     result = function(**params)
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/botocore/client.py", line 357, in _api_call
localstack_1  |     return self._make_api_call(operation_name, kwargs)
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/botocore/client.py", line 634, in _make_api_call
localstack_1  |     api_params, operation_model, context=request_context)
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/botocore/client.py", line 682, in _convert_to_request_dict
localstack_1  |     api_params, operation_model)
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/botocore/validate.py", line 297, in serialize_to_request
localstack_1  |     raise ParamValidationError(report=report.generate_report())
localstack_1  | botocore.exceptions.ParamValidationError: Parameter validation failed:
localstack_1  | Invalid type for parameter tags, value: [{'Key': 'Name', 'Value': 'A-DLQ'}, {'Key': 'Cluster', 'Value': 'AEvents'}], type: <class 'list'>, valid types: <class 'dict'>
localstack_1  |
```

Thanks everyone for any feedback!

**Comment by nakedcity** [comment]
Any update?

**Comment by whummer** [comment]
Thanks for reporting @hikarisakamoto @nakedcity . Has this issue been fixed in the meantime? If not, then can you please provide a small example (CloudFormation config) that helps reproduce this issue? Thanks!

**Comment by whummer** [comment]
Should be fixed in #2114 - can you please give it a try with the `latest` version? Please re-open this issue if the problem persists. Thanks
